### ISSUE TITLE
Spotweb

### ISSUE THREAD

**Issue by hypetrax** [Mar2zz/LaSi#22]
Hi, i'm kinda new into this and things i tried was removing mysql with commands as:
apt-get remove --purge mysql*
apt-get remove autoclean / autoremove

And this error still pops up.

Are you sure you want to continue and install Spotweb?

Packagelist is up to date
2012-02-02 20:48:04 URL:http://dl.dropbox.com/u/18712538//LaSi_Repo/spotweb.deb [1564/1564] -> "/tmp/spotweb.deb" [1]
Selecting previously deselected package spotweb.
(Reading database ... 92398 files and directories currently installed.)
Unpacking spotweb (from /tmp/spotweb.deb) ...
dpkg: dependency problems prevent configuration of spotweb:
 spotweb depends on php5-mysql; however:
  Package php5-mysql is not installed.
 spotweb depends on mysql-server; however:
  Package mysql-server is not installed.
dpkg: error processing spotweb (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 spotweb

Solving dependencys...
Reading package lists... Done
Building dependency tree
Reading state information... Done
Correcting dependencies... Done
The following extra packages will be installed:
  libdbd-mysql-perl libmysqlclient16 mysql-client-5.1 mysql-client-core-5.1
  mysql-common mysql-server mysql-server-5.1 mysql-server-core-5.1 php5-mysql
Suggested packages:
  tinyca mailx
The following NEW packages will be installed:
  libdbd-mysql-perl libmysqlclient16 mysql-client-5.1 mysql-client-core-5.1
  mysql-common mysql-server mysql-server-5.1 mysql-server-core-5.1 php5-mysql
0 upgraded, 9 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
Need to get 0 B/22.9 MB of archives.
After this operation, 58.5 MB of additional disk space will be used.
Preconfiguring packages ...
Selecting previously deselected package mysql-common.
(Reading database ... 92401 files and directories currently installed.)
Unpacking mysql-common (from .../mysql-common_5.1.54-1ubuntu4_all.deb) ...
Selecting previously deselected package libmysqlclient16.
Unpacking libmysqlclient16 (from .../libmysqlclient16_5.1.54-1ubuntu4_amd64.deb) ...
Selecting previously deselected package php5-mysql.
Unpacking php5-mysql (from .../php5-mysql_5.3.5-1ubuntu7.4_amd64.deb) ...
Selecting previously deselected package libdbd-mysql-perl.
Unpacking libdbd-mysql-perl (from .../libdbd-mysql-perl_4.016-1_amd64.deb) ...
Selecting previously deselected package mysql-client-core-5.1.
Unpacking mysql-client-core-5.1 (from .../mysql-client-core-5.1_5.1.54-1ubuntu4_amd64.deb) ...
Selecting previously deselected package mysql-client-5.1.
Unpacking mysql-client-5.1 (from .../mysql-client-5.1_5.1.54-1ubuntu4_amd64.deb) ...
Selecting previously deselected package mysql-server-core-5.1.
Unpacking mysql-server-core-5.1 (from .../mysql-server-core-5.1_5.1.54-1ubuntu4_amd64.deb) ...
Processing triggers for man-db ...
Setting up mysql-common (5.1.54-1ubuntu4) ...
Selecting previously deselected package mysql-server-5.1.
(Reading database ... 92600 files and directories currently installed.)
Unpacking mysql-server-5.1 (from .../mysql-server-5.1_5.1.54-1ubuntu4_amd64.deb) ...
Selecting previously deselected package mysql-server.
Unpacking mysql-server (from .../mysql-server_5.1.54-1ubuntu4_all.deb) ...
Processing triggers for ureadahead ...
Processing triggers for man-db ...
Setting up libmysqlclient16 (5.1.54-1ubuntu4) ...
Setting up php5-mysql (5.3.5-1ubuntu7.4) ...
Setting up libdbd-mysql-perl (4.016-1) ...
Setting up mysql-client-core-5.1 (5.1.54-1ubuntu4) ...
Setting up mysql-client-5.1 (5.1.54-1ubuntu4) ...
Setting up mysql-server-core-5.1 (5.1.54-1ubuntu4) ...
Setting up mysql-server-5.1 (5.1.54-1ubuntu4) ...
120202 20:48:30 [Note] Plugin 'FEDERATED' is disabled.
120202 20:48:30  InnoDB: Initializing buffer pool, size = 8.0M
120202 20:48:30  InnoDB: Completed initialization of buffer pool
120202 20:48:30  InnoDB: Started; log sequence number 0 536113066
120202 20:48:30  InnoDB: Starting shutdown...
120202 20:48:35  InnoDB: Shutdown completed; log sequence number 0 536113066
start: Job failed to start
invoke-rc.d: initscript mysql, action "start" failed.
dpkg: error processing mysql-server-5.1 (--configure):
 subprocess installed post-installation script returned error exit status 1
dpkg: dependency problems prevent configuration of mysql-server:
 mysql-server depends on mysql-server-5.1; however:
  Package mysql-server-5.1 is not configured yet.
dpkg: error processing mysql-server (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of spotweb:
 spotweb depends on mysql-server; however:
  Package mysql-server is not configured yet.
  Package mysql-server-5.1 which provides mysql-server is not configured yet.
dpkg: error processing spotweb (--configure):
 dependency problems - leaving unconfigured
Processing triggers for libc-bin ...
No apport report written because the error message indicates its a followup error from a previous failure.
                          No apport report written because the error message indicates its a followup error from a previous failure.
                                                    ldconfig deferred processing now taking place
Errors were encountered while processing:
 mysql-server-5.1
 mysql-server
 spotweb
E: Sub-process /usr/bin/dpkg returned an error code (1)

Failed! Installing Spotweb didn't finish, try again or:
Copy the text above and report an issue at the following address:
https://github.com/Mar2zz/LaSi/issues

I hope you can tell me what im doing wrong.

**Comment by Mar2zz** [comment]
I dunno why your mysql install is failing. You should use dpkg-reconfigure on that. Did you delete files yourself? mysql shouldn't fail on a 'vanilla' buntu machine which never had mysql before. 

type dpkg-reconfigure --help to see available options. Usually it's sudo dpkg-reconfigure packagename, so ini your case sudo dpkg-reconfigure mysql-server-5.1.

Let me know if you can solve this that way.

**Comment by Mar2zz** [comment]
reopen if not solved yet
